packages/@aws-cdk/toolkit-lib/lib/api/drift/interfaces.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Interface for IO operations used in drift detection
+ */
+export interface IDriftIoHelper {
+  /**
+   * Send a notification
+   */
+  notify(msg: any): Promise<void>;
+}
```

Comment:
I don't see the need for this being different than the regular IoHelper. AI gone rouge?

Reply:
This change will become clearer once the function is moved to the `toolkit-lib` module. We no longer conditionally emit output. Instead, all output is emitted as messages with an appropriate log level. If you need to support additional output when running `cdk drift -v`, you can assign those messages a `debug` log level.

Your code appears to have 4 sections, which I would classify like this:
- unchanged -> debug
- unchecked -> debug
- modified -> result
- deleted -> result